### DISCUSSION TITLE
GVF SPORT - Test Case

### DISCUSSION THREAD

**Discussion by laurizioA** [NASA-LIS/LISF#1289] [Q&A] [answered]
Hello!

Where can I download it to run the GVF SPORT testcase?

this "gvf_SPORT_3KM.MAX.1gd4r" file.

**Comment by dmocko** [comment, marked as answer]
Hi @laurizioA 

We generally only support the public end-to-end testcase, but you can find this particular file in this bundle:

https://portal.nccs.nasa.gov/lisdata_pub/TESTING/testcases/data/7.4/0/sportgvftest_ldt_v74.tar.gz
